rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json

```diff
@@ -0,0 +1,925 @@
+            "type": "string",
+            "minimum": -9999,
+            "maximum": 2050
+          },
+          "date": {
+            "title": "Date of publication (free formed)",
+            "description": "Date of the publication in a free form. If there's a normalized date of publication, then a free formed date can be added to be displayed.",
+            "type": "string"
+          }
+        }
+      }
+    },
+    "extent": {
+      "title": "Extent",
+      "description": "Extent of the resource, ie number of pages or volumes.",
+      "type": "string",
+      "minLength": 1
+    },
+    "otherMaterialCharacteristics": {
+      "title": "Other Material Characteristics",
+      "description": "Other Material Characteristics, ie illustrations, black and with or coloured.",
+      "type": "string",
+      "minLength": 1
+    },
+    "formats": {
+      "title": "Format",
+      "description": "Format of the resource, ie dimensions in cm.",
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "type": "string",
+        "minLength": 1
+      }
+    },
+    "additionalMaterials": {
+      "title": "Additional materials",
+      "description": "Accompanying material of the resource, ie maps.",
+      "type": "string",
+      "minLength": 1
+    },
+    "series": {
+      "title": "Series",
+      "description": "Series to which belongs the resource.",
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "type": "object",
+        "required": [
+          "name"
+        ],
+        "additionalProperties": false,
+        "properties": {
+          "name": {
+            "title": "Title",
+            "description": "Title of the series.",
+            "type": "string"
+          },
+          "number": {
+            "title": "Numbering",
+            "description": "Numbering of the resource within the series.",
+            "type": "string"
+          }
+        }
+      }
+    },
+    "notes": {
+      "title": "Note",
+      "description": "Note on the resource.",
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "type": "string",
+        "minLength": 1
+      }
+    },
+    "abstracts": {
+      "title": "Abstract",
+      "description": "Abstract of the resource.",
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "type": "string",
+        "minLength": 3
+      }
+    },
+    "identifiedBy": {
+      "title": "Identifiers",
+      "description": "Persistent identifiers of the resource, ie reroID and ISBN.",
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "type": "object",
+        "required": [
+          "type",
+          "value"
+        ],
+        "additionalProperties": false,
+        "properties": {
+          "type": {
+            "title": "Type",
+            "description": "Type of the value.",
```

Comment:
Remove this description, less one have a more informative sentence to be added. Or something like `Identifiers type.` But I think it's better to remove it.